Form/DoctrineMongoDBTypeGuesser.php

```diff
@@ -38,13 +38,8 @@ public function __construct(ManagerRegistry $registry)
         $this->registry = $registry;
     }
 
-    /**
-     * @param string $class
-     * @param string $property
-     *
-     * @return TypeGuess|null
-     */
-    public function guessType($class, $property)
+    /** @return TypeGuess|null */
+    public function guessType(string $class, string $property)
```

Comment:
[`FormTypeGuesserInterface::guessType()` already has `string` in 5.4](https://github.com/symfony/symfony/blob/5.4/src/Symfony/Component/Form/FormTypeGuesserInterface.php#L24)